packages/@n8n/nodes-langchain/credentials/OpenRouterApi.credentials.ts

```diff
@@ -0,0 +1,48 @@
+import type {
+	IAuthenticateGeneric,
+	ICredentialTestRequest,
+	ICredentialType,
+	INodeProperties,
+} from 'n8n-workflow';
+
+export class OpenRouterApi implements ICredentialType {
+	name = 'openRouterApi';
+
+	displayName = 'OpenRouter';
+
+	documentationUrl = 'openrouter';
+
+	properties: INodeProperties[] = [
+		{
+			displayName: 'API Key',
+			name: 'apiKey',
+			type: 'string',
+			typeOptions: { password: true },
+			required: true,
+			default: '',
+		},
+		{
+			displayName: 'Base URL',
+			name: 'url',
+			type: 'string',
+			default: 'https://openrouter.ai/api/v1',
+			description: 'Override the default base URL for the API',
```

Comment:
This isn't really overridable, right?

Reply:
Ah, I didn't realize there's hidden type. Thanks!